setup.py

```diff
@@ -18,11 +18,11 @@
 
 setup(
     name="mod-net",
-    version="v2.0.0-beta3",
+    version="v2.0.0-beta4",
     description="Make SET and SEU fault injections in hierarchical verilog netlists",
     packages=find_packages(exclude=("tests",)),
     install_requires=[
-        'Click',
+        'Click>=7.1.2',
```

Comment:
Yes, that is true. But you can’t be sure your program will working with the version 9 of click or a version lower than 7. You can put a ==7.* and you will be sure this will work because click maybe will fix some issues or add some fetures but dont will be change the implementation.
```suggestion
        'Click==7.*',
```